spec/services/make_an_offer_spec.rb

```diff
@@ -38,4 +38,37 @@
       expect(decision).not_to be_valid
     end
   end
+
+  describe 'decline by default' do
+    def create_application
+      application_form = create :completed_application_form
+      create(
+        :application_choice,
+        application_form: application_form,
+        status: 'awaiting_provider_decision',
+        edit_by: 2.business_days.ago,
+        )
+    end
+
+    it 'sets the `offered_at` date' do
```

Comment:
Does this `it` belong in the `decline by default` context?

Reply:
Not sure I got this, is not aready in there?

```ruby
describe 'decline by default' do
 ...
  it 'sets the `offered_at` date' do
  end
```

Reply:
I think it shouldn’t be, it’s not really part of decline by default.

Reply:
👍 
I got it! at first, in my mind was kind of related.
refactored